seqr/views/utils/permissions_utils.py

```diff
@@ -19,14 +20,50 @@ def get_project_and_check_permissions(project_guid, user, **kwargs):
     return project
 
 
-def has_project_permissions(project, user, can_edit=False, is_owner=False):
+def is_staff(user, session):
+    """
+    Background of this function.
+
+    The staff management with an AnVIL group is problematic because it hard to tracking the changes in the group.
+    Since we keep user models on seqr, the 'is_staff' is available from the model.
+    """
+    return user.is_staff
```

Comment:
Based on the comment you wrote, this function should not exist at all, and its just confusing. Is there a reason you kept it?

Reply:
No. It is going to be removed.